src/style.css

```diff
@@ -1 +1,75 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+body {
+  background: #e5e5e5;
+  margin: 0;
+}
+
+:root {
+  --active-hover-color: #00acdc;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  background: #fff;
+  justify-content: flex-end;
```

Comment:
```suggestion
  justify-content: space-between;
```
make it work

Reply:
also add horizontal padding for header (50 px) and remove right margin for `header__list`

Reply:
<img width="1440" alt="Screenshot 2023-02-09 at 10 27 33" src="https://user-images.githubusercontent.com/94289038/217758653-0e5eab56-91f1-4727-a17f-30323f125bc1.png">